tests/test_metric.py

```diff
@@ -25,8 +28,8 @@ def mock_zeus_monitor():
     with patch("zeus.metric.ZeusMonitor", autospec=True) as zeus_monitor:
         mock_instance = zeus_monitor.return_value
         mock_instance.end_window.return_value = MagicMock(
-            gpu_energy={0: 30.0, 1: 35.0, 2: 40.0},
-            cpu_energy={0: 20.0, 1: 25.0},
+            gpu_energy={0: 50.0, 1: 100.0, 2: 200.0},
+            cpu_energy={0: 40.0, 1: 50.0},
             dram_energy={},
```

Comment:
If mock CPU 0 supports DRAM energy measurement (in `mock_get_cpus`), shouldn't this be something like `dram_energy={0: 10.0}`?